site/docs/numbers/porting/updateBulkPortins.mdx

```diff
@@ -0,0 +1,220 @@
+---
+id: updateBulkPortins
+title: Update or Delete Bulk Port-in
+slug: /numbers/guides/porting/updateBulkPortins
+description: How to update bulk port-in order using the Bandwidth API
+keywords:
+  - bandwidth
+  - bulkportins
+  - portin
+image: '@site/static/img/bw-icon.svg'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+Follow this guide if you'd like to update or delete a Bulk Port-in order.
+
+## Update Bulk Port-in
+
+It is possible to update Bulk Port-in order using ```PUT``` or ```PATCH``` request to ```/bulkPortins/{orderId} API``` on the existing order-id:
+* If Bulk Port-in is not in terminal (```COMPLETED```, ```CANCELLED```, or ```PARTIAL```) status
+* As with the ```POST```, any data associated with the bulk port-in will cascade to subtending orders when they are created
+* If the port-in orders contained within the Bulk Port are in ```DRAFT``` state, any field can be modified
+* If any child port-in order in the Bulk Port-in is in any other state, normal SUPP rules apply, and the list of appropriate fields is smaller
+
+:::info
+Subtending orders are created after tns are added to the bulk port-in using the /tnList endpoint, see the next section of this tutorial.
```

Comment:
Change "tns" to "telephone numbers"

Reply:
Changed. We have also TNs in different sections in the doc, is it ok?

Reply:
We'll just leave those others for now and just change this instance.